tools/packages/wesl/src/parse/WeslGrammar.ts

```diff
@@ -158,7 +158,7 @@ const attribute_argument_list = delimited(
   "(",
   withSep(
     ",",
-    span(fn(() => expression))     .collect(expressionCollect, "attrParam"), // TODO: These unknown expressions have decls inside of them, that's why they're tough to replace!
+    span(fn(() => expression))     .collect(expressionCollect, "attrParam"), // LATER These unknown expressions have decls inside of them, that's why they're tough to replace!
```

Comment:
This will become relevant whenever I start parsing the entire syntax tree. I think I'll end up doing that sooner or later.